src/app/router.options.ts

```diff
@@ -1,18 +1,24 @@
 import type { RouterConfig } from '@nuxt/schema'
 
 export default <RouterConfig> {
-  // https://router.vuejs.org/api/interfaces/routeroptions.html#routes
-  // alternatively, could put this inside the setup for each page
   routes: _routes => [
     {
       name: RouteNameE.DASHBOARD,
-      path: '/:identifier',
+      path: '/:identifier?',
       component: () => import('~/pages/dashboard.vue').then(r => r.default || r),
       meta: {
         layout: 'business',
         title: 'Business Dashboard',
         breadcrumbs: [getBcrosHomeCrumb, getRegistryDashCrumb, getBusinessDashCrumb],
         staffBreadcrumbs: [getStaffDashCrumb, getBusinessDashCrumb]
+      },
+      beforeEnter(to, from, next) {
+        // If identifier is missing
+        if (!to.params.identifier) {
+          next({ name: RouteNameE.DASHBOARD, params: { identifier: 'default_identifier' } })
```

Comment:
Clever solution!

Instead of setting a "special value", could you reroute to a (new) error page directly?

Reply:
After testing this, it looks fine. Resolving.